PayKids/presentation/src/main/java/com/paykids/presentation/view/mypage/ModifyInfoFragment.kt

```
@@ -0,0 +1,86 @@
+import com.paykids.presentation.base.BaseFragment
+import com.paykids.presentation.databinding.FragmentModifyInfoBinding
+import com.paykids.presentation.utils.UiState
+import com.paykids.presentation.view.signIn.SignActivity
+import dagger.hilt.android.AndroidEntryPoint
+
+@AndroidEntryPoint
+class ModifyInfoFragment : BaseFragment<FragmentModifyInfoBinding>() {
+    private val myPageViewModel: MyPageViewModel by viewModels()
+
+    override fun initView() {
+    }
```

detekt (1.23.7)

[warning] This empty block of code can be removed.

(detekt.empty-blocks.EmptyFunctionBlock)